src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/esp/modes/Esp2DMode.kt

```diff
@@ -72,18 +80,48 @@ object Esp2DMode : EspMode("2D") {
                 val minY = projected.minOf { it.y }
                 val maxY = projected.maxOf { it.y }
                 val minZ = projected.minOf { it.z } // TODO: Handle Z-index correctly
-                val rectWidth = (maxX - minX).toInt()
-                val rectHeight = (maxY - minY).toInt()
+                var rectWidth = (maxX - minX)
+                var rectHeight = (maxY - minY)
+                val black = 0xFF000000.toInt()
```

Comment:
Define properly. `0xFF000000.toInt()` is not readable. Use `Color4b.BLACK.toARGB()` instead.